### PR TITLE
Adding is_alive method and py3 support

### PR DESCRIPTION
**Pull request by GGabriele**

### CODE REVIEW

**Comment by dbarrosop** [comment]
Looks good to me. Feel free to merge, add labels and release when you see fit : )

File: napalm_panos/panos.py [hunk 1]
```diff
@@ -154,6 +154,12 @@ def _import_file(self, filename):
         else:
             return path
 
+    def is_alive(self):
+        if self.device:
```
Comment (mirceaulinic) [inline napalm_panos/panos.py]: If the connection is established via SSH, you should probably return the connection status.

Comment (GGabriele) [inline napalm_panos/panos.py]: You're right...I fixed it

**Comment by GGabriele** [comment]
@mirceaulinic have you got the chance to review this?

**Comment by mirceaulinic** [comment]
Sorry... dropped the ball here

**Pull request merged**